token/cli/src/main.rs

```diff
@@ -254,6 +255,14 @@ fn command_create_token(
             decimals,
         )?,
     ];
+
+    // check for memo text; if it exists convert it to
+    // bytes and add memo tx instructions
+    if let Some(text) = memo {
+        let memo_instruction = spl_memo::build_memo(text.as_bytes(), &[&config.owner]);
+        instructions.push(memo_instruction);
```

Comment:
how about a one-liner here:
```suggestion
        instructions.push(spl_memo::build_memo(text.as_bytes(), &[&config.owner]));
```